openapi/googleapis.retail/Ballerina.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 license = ["Apache-2.0"]
-keywords = ["sales", "paid"]
+keywords = ["Commerce/eCommerce/", "Cost/Paid", "Vendor/Google"]
```

Comment:
The last `/` should removed from "Commerce/eCommerce/"

Reply:
Removed "/" at the end